api/package.json

```diff
@@ -33,24 +39,30 @@
     "compression": "^1.7.4",
     "cors": "^2.8.5",
     "cross-env": "^5.2.0",
-    "dotenv": "^6.0.0",
+    "dotenv": "^6.2.0",
+    "drizzle-orm": "^0.29.3",
+    "drizzle-zod": "^0.5.1",
     "express": "^4.18.2",
     "helmet": "^7.1.0",
     "lodash": "^4.17.10",
     "loopback": "^3.25.1",
     "loopback-boot": "^3.3.0",
     "loopback-component-explorer": "^6.0.0",
-    "loopback-connector-mysql": "^5.2.0",
+    "loopback-connector-mysql": "^7.0.6",
```

Comment:
upgraded this connector. so we no longer have to disable our local db to connect :)